src/main/kotlin/no/nav/familie/ef/sak/vilkår/VurderingService.kt

```diff
@@ -121,16 +121,22 @@ class VurderingService(
         val personIdent = behandlingService.hentAktivIdent(behandlingId)
         val barn = barnService.finnBarnPåBehandling(behandlingId)
         val grunnlag = vilkårGrunnlagService.hentGrunnlag(behandlingId, søknad, personIdent, barn)
+
         val søktOmBarnetilsyn =
             grunnlag.barnMedSamvær.filter { it.barnepass?.skalHaBarnepass == true }.map { it.barnId }
+        val harBrukerEllerAnnenForelderTidligereVedtak =
+            grunnlagsdataService.hentGrunnlagsdata(behandlingId).grunnlagsdata.annenForelder.any {
+                it.tidligereVedtaksperioder.tilDto().harTidligereVedtaksperioder()
+            } || grunnlag.tidligereVedtaksperioder.harTidligereVedtaksperioder()
```

Comment:
Kanskje kan trekke ut dette til en egen metode :) 
Betyder dette at det kun kan automatiskt godkjennes for førstegangsbehandlinger?

Reply:
Trenger egentlige ikke å hente grunnlaget på nytt
```kotlin
val harBrukerEllerAnnenForelderTidligereVedtak =
            grunnlag.barnMedSamvær.mapNotNull { it.søknadsgrunnlag.forelder?.tidligereVedtaksperioder }
                .any { it.harTidligereVedtaksperioder() }
            || grunnlag.tidligereVedtaksperioder.harTidligereVedtaksperioder()
```